utils/near-cache/src/lib.rs

```diff
@@ -0,0 +1,57 @@
+use cached::{Cached, SizedCache};
+use std::hash::Hash;
+use std::sync::Mutex;
+
+#[allow(unused)]
+struct MyCache<K, V> {
+    inner: Mutex<SizedCache<K, V>>,
+}
+
+// MyCache<CryptoHash, Result<wasmer_runtime::Module, CompilationError>>
+impl<K, V> MyCache<K, V>
+where
+    K: Hash + Eq + PartialEq + Clone,
+    V: Clone,
+{
+    #[allow(unused)]
+    pub fn new(capacity: usize) -> Self {
```

Comment:
rustc complains because `MyCache` is private, so the compiler knows that there can't be any usges. A more natural way to fix this would be to make `MyCache` a `pub` struct.